### PR TITLE
 image-source: Add all available transitions to slideshow source

### PR DESCRIPTION
**Pull request by cg2121**
~~Still one problem: I can't figure out why it crashes on exit.~~

### CODE REVIEW

**Comment by jp9000** [comment]
Hm, seems this is getting some undefined symbols on mac.  Do you have a mac for testing, @cg2121?  If not I'll help out when I can get around to it.

**Comment by cg2121** [comment]
No, I don't have a Mac.

**Comment by RytoEX** [comment]
I'm wondering if it's a good idea to entwine image-source with the frontend-api like this.  I might be worrying about nothing, but it's gnawing at me.

**Comment by jp9000** [comment]
I don't know if this is the correct approach.  I don't want the image-source plugin to depend on obs-frontend-api.  Do you think you could try enumerating existing transitions?  Hm, I definitely don't want this plugin to depend on obs-frontend-api if possible.  Or, I would be okay with obs-frontend-api, but only if it was strictly a cmake option, disabled by default, otherwise it'd have its default functionality from before.

**Comment by cg2121** [comment]
I have now added the cmake variable for the frontend api.

File: plugins/image-source/CMakeLists.txt [hunk 1]
```diff
@@ -14,6 +14,12 @@ add_library(image-source MODULE
 	${image-source_SOURCES})
 target_link_libraries(image-source
 	libobs
+	obs-frontend-api
```
Comment (jp9000) [inline plugins/image-source/CMakeLists.txt]: You're including it twice here.

**Comment by cg2121** [comment]
I have now fixed the tab indent in the cmake file.

**Comment by jp9000** [comment]
I've changed my mind about this.  Although it would be neat to be able to have access to more transitions, I think having the existing curated list specific to the slideshow is perfectly fine.  I don't think we need to go all feature crazy on something like the slide show.

**Pull request closed without merging**